packages/simcore-sdk/src/simcore_sdk/node_ports_v2/port_utils.py

```diff
@@ -0,0 +1,106 @@
+import logging
+import shutil
+from pathlib import Path
+from typing import Any, Callable, Coroutine, Dict, Optional
+
+from ..node_ports import config, data_items_utils, filemanager
+from .links import DownloadLink, FileLink, ItemConcreteValue, PortLink
+
+log = logging.getLogger(__name__)
+
+
+async def get_value_from_link(
+    key: str,
+    value: PortLink,
+    fileToKeyMap: Optional[Dict[str, str]],
+    node_port_creator: Callable[[str], Coroutine[Any, Any, Any]],
+) -> ItemConcreteValue:
+    log.debug("Getting value %s", value)
+    # create a node ports for the other node
+    other_nodeports = await node_port_creator(value.node_uuid)
+    # get the port value through that guy
+    log.debug("Received node from DB %s, now returning value", other_nodeports)
+
+    value = await other_nodeports.get(value.output)
```

Comment:
yes it can raise anything that the node_ports usually raises. but this should be handled by the client using the node_ports. that would mean some broken link in the pipeline.